### ISSUE TITLE
Tank eating up resonant crystal liquid from smelter

### ISSUE THREAD

**Issue by DestinyPlayer** [McJty/DeepResonance#70]
Basically, when my smelter finishes up smelting the ore, it doesn't appear in the tank above. Sometimes, after breaking and replacing the whole construction several times, it works, but then it breaks once again after some time. What's the reason?

[Deep Resonance Log.txt](https://github.com/McJty/DeepResonance/files/253654/Deep.Resonance.Log.txt)

**Comment by The-Fireplace** [comment]
Yeah, using The One Probe, I checked the liquid level before and after smelting 2 ore, and it stayed at 600B.

**Comment by McJty** [comment]
What version of Deep Resonance? Also can you show a screenshot

**Comment by The-Fireplace** [comment]
I had this issue at one point, and commented on it today, but decided to update Forge, then go back and test to see if I could pinpoint the issue, and wasn't able to reproduce it. Anyone who thinks they may be having this issue with the 1.9.4 version, I suggest updating Forge and seeing if that fixes it.

**Comment by cj89898** [comment]
I still have this issue. Here is my exact setup: https://gyazo.com/9af3f454467f86f5f8d5139ad07ce95f
I'm not sure how to reproduce it exactly, but I do know that it keeps happening. When I get 6 buckets of RCL I use ender IO conduits to pump it out into another tank. I went back to make another crystal, set resonating ore in the smelter and it created 0 RCL with 8 buckets of lava beneath it. I wasted 25 before even realizing it. The RCL renders fine in the tank, but there is none there. I had to break and replace the tank for it to work.

Minecraft Version: 1.9.4
Deep Resonance Version: 1.9.4-1.2.0beta35
Forge version: 1.9.4-12.17.0.1968

**Comment by McJty** [comment]
Hmm I wonder if this is an interaction bug with EnderIO. Does it always work fine if you don't use EnderIO?

**Comment by cj89898** [comment]
Oh I forgot to put that in my comment, I have no other way to transport liquid without Ender IO. :L

**Comment by McJty** [comment]
yes but at least you can see if the RCL gets generated. If it does then put back the connector to see where it goes. Just as a test mind you

**Comment by cj89898** [comment]
Oh you want me to just remove and test, I will do it now.

**Comment by cj89898** [comment]
Okay I tried it without the connector and the same thing happened. https://gyazo.com/eada24f762882b3be7696d62bd5d1465 If I broke and replaced the tank, it would work fine so maybe it's the Ender IO conduit breaking it when it removes all the liquid in it?

**Comment by McJty** [comment]
For better debugging purposes you could use The One Probe together with waila. The One Probe actually shows the exact contents as it is in the server. So it is more accurate in that sense

**Comment by cj89898** [comment]
I do have that mod installed. Give me just one second. When I break the tank nothing is in it though. I'll try to replicate the issue.

**Comment by Wrsaith** [comment]
I'm getting this same issue using 1.12.2-forge1.12.2-14.23.5.2846 
deepresonance-1.12-1.8.0

and it just randomly starts dropping it in after like 2 full stacks of resonant have been eaten before outputting anyting to a tank.

and no i can' not upgrade forge at this time. running the current FTB revelations modpack.

**Comment by Daniel-AmaFa** [comment]
Same issue in minecraft 1.16.5 - forge 36.2.39 - mod version 1.16-2.0.11

I tried with multiple tanks from various mods, the smelter seems to detect the tanks since it does start operation, but it's only consuming the ore and not generating anything.

It problably has to do with the smelter not generating/dumping the liquid crystal (?

**Comment by McJty** [comment]
Is the tank below the smelter approx 50% filled with Lava? That's VERY important. Otherwise the ores will be wasted

**Comment by Xlaits** [comment]
Running ATM9 and also having this same issue.

**Comment by Xlaits** [comment]
ATM9 also has this issue. Tanks are set up correctly, even when being fed plenty of Lava, it's just voiding the Liquid Crystal. Mod is almost entirely unusable due to this, and I do quite enjoy the concept...

**Comment by McJty** [comment]
'Fed plenty of lava'? I hope you only filled it 50%? If you keep a tank full of lava it will consume the liquid crystal

**Comment by Xlaits** [comment]
Why... why is that a requirement that it be so meticulously monitored? Not a single other mod I am aware of limits **the maximum fuel** you can use in their specific furnaces, while also still requiring power. None of the pipe mods that I am aware of can allow for specific amounts to be pushed through and then stop, per percentage. Nowhere am I told that key bit of information, either, that I am currently aware of, especially playing in a modpack like ATM9.

EDIT:
I went through the quest line for DeepResonance on ATM9 and yes, there is a single, throwaway line mentioning keeping the lava "between 4 to 6 buckets" but that's it.

This mod would benefit from an in-game instruction manual...

**Comment by McJty** [comment]
I have no control over the quests so can't comment on that. 
The tooltip of the smelter actually says this:

_This machine smelts resonating ore and produces liquid crystal in a tank placed on top of this. Place a tank below the smelter about half-filled with lava_

**Comment by Xlaits** [comment]
Still a throwaway line that would be easily missed, as in the fact that this entire issue exists.

**Comment by McJty** [comment]
Isn't patchouli included with the modpack? Because Deep Resonance actually has an in game guide which is enabled if patchouli is there

**Comment by Xlaits** [comment]
Patchouli is installed, but it doesn't seem like there's any way to access the in game guide. If there's supposed to be a book or something, it isn't showing up.

**Comment by McJty** [comment]
That guide is called the 'Technology Guide' btw

**Comment by McJty** [comment]
Also you can access that guide by pressing the '?' button on the right of Deep Resonance user interfaces

**Comment by Xlaits** [comment]
Yeah, it isn't opening, and the only "Technology Guide" is added by RFTools.

**Comment by McJty** [comment]
Yes, Deep Resonance is part of the RFTools suite. So it also includes it manual in that guide.
Strange that it isn't working though. I'll have to check that

**Comment by Xlaits** [comment]
![Image](https://github.com/user-attachments/assets/59c884a0-f3be-4a91-91df-088d9c9a1805)

**Comment by McJty** [comment]
yes, I also just realized that I never finished writing the deep resonance manual fully. Oops. So yes, the tooltips of the machines are currently your only resource (or external resources like YouTube walkthroughs). Sorry for that
